homeassistant/components/foscam/strings.json

```
@@ -11,7 +11,12 @@
           "stream": "Stream"
         },
         "data_description": {
-          "host": "The hostname or IP address of your Foscam camera."
+          "host": "The hostname or IP address of your Foscam camera.",
+          "port": "The port of your Foscam camera, default is 88.",
+          "username": "The username to login to your Foscam camera.",
+          "password": "The password to login to your Foscam camera.",
```

"to log in" is a verb here:

```suggestion
          "username": "The username to log in to your Foscam camera.",
          "password": "The password to log in to your Foscam camera.",
```

---

Modified, thank you for the feedback!